test/coverband/reporters/base_test.rb

```diff
@@ -47,45 +46,4 @@ class ReportsBaseTest < Minitest::Test
 
     assert_equal expected, Coverband::Reporters::Base.send(:get_current_scov_data_imp, store, roots)[:merged]
   end
-
-  ###
-  # This test uses real world example data which helped uncover a bug
-  # The copied data doesn't format easily and isn't worth the effort to meet
-  # string style
-  # rubocop:disable all
-  ###
-  test '#get_current_scov_data_imp merges multiples of file data' do
```

Comment:
This test was failing with missing mock expectations. I struggled with it for a while and thought maybe it wasn't worth maintaining. Let me know what you think.

Reply:
I guess if this blocks the PR comment out and I will take a look and see if I can quickly fix or I will remove

Reply:
Ok sounds good.